lending/fixtures/workflow.json

```diff
@@ -0,0 +1,167 @@
+[
+ {
+  "docstatus": 0,
+  "doctype": "Workflow",
+  "document_type": "Loan Application",
+  "is_active": 1,
```

Comment:
Some more thoughts to gradually roll out changes, let's disable the workflow by default. 
Users can enable as per their requirement